### DISCUSSION TITLE
Winget version is not the latest

### DISCUSSION THREAD

**Discussion by pillash** [gitextensions/gitextensions#11599] [Q&A]
### Environment

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

### Issue description

In the main Github README there is a [link to Winget](https://winget.run/pkg/GitExtensionsTeam/GitExtensions) to install GitExtensions, but the version on Winget is 4.0.2, not the latest 4.2.1. Can winget be updated with the latest version, or if it is no longer a supported installation method, can the Winget link be removed?

### Steps to reproduce

Go to https://winget.run/pkg/GitExtensionsTeam/GitExtensions to see latest version on winget

**Comment by gerhardol** [comment]
@gitextensions/git-extensions-team Who is maintaining the Winget package?

**Reply by RussKie** [reply]
We do not manage winget nor appget nor chocolatey packaging, those are done by external maintainers.

**Comment by as-marcb** [comment]
The whole `winget.run` site seems to have stopped updating its contents, the actual repository seems fine.
The Git version there also is from [January 2023](https://github.com/git-for-windows/git/releases/tag/v2.39.2.windows.1), despite the actual [package registry](https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/Git/Git) receiving steady updates.

So removing the link to a stale/[discontinued](https://github.com/winget-run/web/issues/98#issuecomment-1030319779) site from the [README](https://github.com/gitextensions/gitextensions/blob/master/README.md#downloads) might be prudent.

**Comment by nnfhhvbb** [comment]
> ### Environment
> 
> - GitExtensions version:
> - GIT version: 
> - OS version: 
> - .NET version: 
> 
> 
> ### Issue description
> 
> In the main Github README there is a [link to Winget](https://winget.run/pkg/GitExtensionsTeam/GitExtensions) to install GitExtensions, but the version on Winget is 4.0.2, not the latest 4.2.1. Can winget be updated with the latest version, or if it is no longer a supported installation method, can the Winget link be removed?
> 
> ### Steps to reproduce
> 
> Go to https://winget.run/pkg/GitExtensionsTeam/GitExtensions to see latest version on winget
> 
> ### Did this work in previous version of GitExtensions?
> 
> _No response_
> 
> ### Diagnostics
> 
> _No response_

**Comment by namtab00** [comment]
[My PR](https://github.com/microsoft/winget-pkgs/pull/179204) for the winget v5 update has been manually validated by the winget team.

The v5 for GitExtensions should now be available via winget.
